admin/src/main/scala/io/buoyant/admin/Admin.scala

```diff
@@ -114,7 +115,7 @@ class Admin(val address: SocketAddress) {
         log.debug(s"admin: $url => ${service.getClass.getName}")
         muxer.withHandler(url, service)
     }
-    notFoundView.andThen(muxer)
+    HeadFilter.andThen(notFoundView.andThen(muxer))
```

Comment:
I'm pretty sure filter composition is associative so this doesn't really matter but it's more common to do `A.andThen(B).andThen(C)` instead of `A.andThen(B.andThen(C))`